Create ui-c8y-1021-22-53-tracking-view-in-device-management-seems-to-be-broken-multiple-issues.md (#3095)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1021-22-53-tracking-view-in-device-management-seems-to-be-broken-multiple-issues.md

---
date: ""

> title: Tracking view in Device Management seems to be broken multiple issues

Reviewer: ```suggestion
title: Issues in the Tracking tab in the device details have been fixed
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-4358
version: 1021.22.53
---

> Tracking view in Device Management seems to be broken multiple issues

Reviewer: ```suggestion
In the **Tracking** tab in the device details the device position was not updated in realtime even though the **Realtime** button was activated. This has been fixed and now the device position and the tracking path are both updated in realtime. Moreover, changing the time range and refreshing the list had no effect. This has been fixed and users can again filter events by different time ranges.
```
